types/three/examples/jsm/postprocessing/RenderPass.d.ts

```diff
@@ -1,13 +1,28 @@
-import { Scene, Camera, Material, Color } from '../../../src/Three';
-
+import {
+    Scene,
+    Camera,
+    Material,
+    Color,
+    WebGLMultipleRenderTargets,
+    WebGLRenderTarget,
+    WebGLRenderer,
+} from '../../../src/Three';
 import { Pass, FullScreenQuad } from './Pass';
 
 export class RenderPass extends Pass {
-    constructor(scene: Scene, camera: Camera, overrideMaterial?: Material, clearColor?: Color, clearAlpha?: number);
-    scene: Scene;
-    camera: Camera;
-    overrideMaterial: Material;
-    clearColor: Color;
+    constructor(scene?: Scene, camera?: Camera, overrideMaterial?: Material, clearColor?: Color, clearAlpha?: number);
+    scene?: Scene;
+    camera?: Camera;
```

Comment:
Can you provide an example or show in the code where `scene` or `camera` can be optional/undefined?

Reply:
I'm just concerned about the properties becoming optional since that would cause any access of those properties to assert that they're not `undefined` or check for it. I would feel better if there were some official three.js examples that show this pattern.